ansible/roles/common/tasks/setfacts.yml

```diff
@@ -177,6 +177,41 @@
   tags:
     - setfacts
 
+- name: set ubuntu-24.04 variables - overrides
+  set_fact:
+    tomcat: "{{ tomcat | default('tomcat10') }}"
+    tomcat_user: "{{ tomcat_user | default('tomcat') }}"
+  when: ansible_os_family == "Debian" and ansible_distribution_version == "24.04"
+  tags:
+  - setfacts
+
+- name: set ubuntu-24.04 variables
+  set_fact:
+    libtcnative: "libtcnative-1"
+    tomcat_package: "{{ tomcat }}"
+    tomcat_user: "{{ tomcat_user }}"
+    tomcat_conf: "/etc/default/{{ tomcat }}"
+    tomcat_conf_dir: "/etc/{{ tomcat }}"
+    tomcat_log_dir: "/var/log/{{ tomcat }}"
+    apache: "apache2"
+    nginx: "nginx"
+    postfix: "postfix"
+    mysql_service: "mysql"
+    mysql: "mysql-server-8.0"
+    cassandra_yaml: "/etc/cassandra/cassandra.yaml"
+    cassandra_user: "cassandra"
+    python_software_properties: "python3-software-properties"
+    tomcat_dir: "/var/lib/{{ tomcat }}"
+    tomcat_webapps: "/var/lib/{{ tomcat }}/webapps/"
+    psycopg2_package: "python3-psycopg2"
+    pg_version: "{{ pg_version | default('16') }}"
+    postgis_version: "{{ postgis_version | default('3') }}"
+    postgis_version_folder: "{{ postgis_version_folder | default('3') }}"
+    postgis_scripts_version: "{{ postgis_scripts_version | default('3.4') }}"
```

Comment:
I think that this should be `3`:

```
$ apt search postgis | grep scripts

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

postgresql-16-pgrouting-scripts/noble 3.6.1-1 all
  Routing functionality support for PostgreSQL/PostGIS - SQL scripts
postgresql-16-postgis-3-scripts/noble 3.4.2+dfsg-1ubuntu3 all
  Geographic objects support for PostgreSQL 16 -- SQL scripts
postgresql-pgrouting-scripts/noble 3.6.1-1 all
  Routing functionality support for PostgreSQL/PostGIS - SQL scripts metapackage
postgresql-postgis-scripts/noble 3.4.2+dfsg-1ubuntu3 all
  Geographic objects support for PostgreSQL -- SQL scripts metapackage
```

Reply:
You mean `postgis_scripts_version` default should be `3` ?  It seems like the relevant version is `3.4.2` for `noble`...